src/main/scala/dcos/metronome/MetronomeConfig.scala

```diff
@@ -110,6 +129,27 @@ class MetronomeConfig(configuration: Configuration) extends JobsConfig with ApiC
 
   override lazy val maxActorStartupTime: FiniteDuration = configuration.getFiniteDuration("metronome.akka.actor.startup.max").getOrElse(10.seconds)
 
+  lazy val metricsNamePrefix: String = configuration.getOptional[String]("metronome.metrics.prefix").getOrElse("metronome")
+
+  lazy val metricsPrometheusReporter: Boolean = configuration.getOptional[Boolean]("metronome.metrics.reporter.prometheus").getOrElse(false)
+  lazy val metricsStatsDReporter: Boolean = configuration.getOptional[Boolean]("metronome.metrics.reporter.statsd").getOrElse(false)
+  lazy val metricsDataDogReporter: Boolean = configuration.getOptional[Boolean]("metronome.metrics.reporter.datadog").getOrElse(false)
+
+  lazy val metricsHistogramReservoirSignificantDigits: Int = configuration.getOptional[Int]("metronome.metrics.histogram.reservoir.significant.digits").getOrElse(3)
```

Comment:
what do you think about not duplicating the defaults here and instead relying on the defaults in Marathon? Or do you think it's best to have those duplicated and if needed change it in both places?

Reply:
yeah.. to use the marathon defaults means to not provide the configuration in metronome... I did think about conditionally providing flags / values...  I'll add in support for it... it will provide value in other ways.